cpu/esp32/include/gpio_arch.h

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (C) 2018 Gunar Schorcht
+ * Copyright (C) 2021 Gunar Schorcht
```

Comment:
```suggestion
 * Copyright (C) 2022 Gunar Schorcht
```

?

Reply:
The PR is from 2021 with some changes after the rebase.